src/layouts/Layout.astro

```diff
@@ -12,6 +12,14 @@ export interface Props {
 }
 
 const { title, description, opengraph } = Astro.props;
+
+const structuredData = JSON.stringify({
+    "@context": "https://schema.org",
+    "@type": "WebSite",
+    name: "Vencord",
+    alternateName: "VC",
+    url: Astro.site,
+});
```

Comment:
is this really necessary here? It seems obsolete as this info is already obtainable via og tags

Reply:
surprisingly, yes! if you have it in open graph and structured data it increases google's reliability at actually detecting your site information (because at the minute it doesnt know the site is called "Vencord", its just "vencord.dev")